ccd-definition/CaseEvent.json

```diff
@@ -309,10 +310,10 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "uploadDocumentsAfterSubmission",
+    "ID": "uploadStandardDirections",
     "Name": "Documents",
-    "Description": "Upload documents after submission",
-    "DisplayOrder": 1,
+    "Description": "Upload standard directions",
+    "DisplayOrder": 2,
```

Comment:
I have reorganised here - uploadDocumentsAfterSubmission is LA event so put it above others and modified the display Order but still waiting for confirmation from Eilis.